addons/website_slides/views/website_slides_templates_homepage.xml

```diff
@@ -502,7 +502,7 @@
 <template id='slides_misc_user_image' name="User Avatar">
     <img t-att-class="img_class if img_class else 'rounded-circle float-left'"
         t-att-style="img_style if img_style else 'width: 32px; height: 32px; object-fit: cover;'"
-        t-att-src="'/profile/avatar/%s?field=image_128' % user.id"
+        t-att-src="'/web/image/res.users/%s/avatar_128' % user.id"
```

Comment:
need to keep using /profile/avatar for access right issue (adapted to be using avatar_128)